eclass/toolchain-funcs.eclass

```diff
@@ -534,10 +534,9 @@ tc-ld-force-bfd() {
 	ewarn "Forcing usage of the BFD linker"
 
 	# Set up LD to point directly to bfd if it's available.
-	local ld=$(tc-getLD "$@")
-	# We need to extract the first word in case there are flags appended
-	# to its value (like multilib), bug #545218.
-	local bfd_ld="${ld%% *}.bfd"
+	# Unset LD first so we get the default value from tc-getLD.
+	local ld=$(unset LD; tc-getLD "$@")
```

Comment:
quotes around `$()` missing (both are correct, but I believe that's our style here?)

Reply:
I don't see quotes being used consistently in this eclass, but I can add them if others think it looks nicer.